webpack.config.js

```diff
@@ -64,6 +65,35 @@ const stylesTransform = ( content ) => {
 	return content;
 };
 
+const blockNameRegex = new RegExp( /(?<=src\/).*(?=(\/frontend))/g );
+
+const createEntrypoints = () => {
+	const scriptPaths = fastGlob.sync(
+		'./packages/block-library/src/**/frontend.js',
+		{
+			ignore: [ '**/build*/**' ],
+		}
+	);
+
+	const scriptEntries = scriptPaths.reduce( ( entries, scriptPath ) => {
+		const [ blockName ] = scriptPath.match( blockNameRegex ) || [];
+
+		return {
+			...entries,
+			[ blockName ]: scriptPath,
```

Comment:
Using the `blockName` as the key may cause interesting issues in the future, if a block has the same name as a package, since `packageEntries` and `scriptEntries` are merged below, any duplicate named entries in `scriptEntries` will override the entry in `packageEntries`.

Reply:
Good catch. At the moment we have a single script entry so it isn't an issue but with the number of packages and blocks that exist, it's possible that they could use the same name. It should be further investigated.